config/environments/production.rb

```diff
@@ -1,6 +1,10 @@
 Otwarchive::Application.configure do
   # Settings specified here will take precedence over those in config/environment.rb
 
+  local_config = YAML.load_file(Rails.root.join("config/local.yml"))
+
+  config.hosts = local_config["PERMITTED_HOSTS"]
```

Comment:
Should this be `ArchiveConfig.PERMITTED_HOSTS` now? I wasn't sure, so I left it alone.